spark/src/main/scala/org/apache/mahout/sparkbindings/blas/ABt.scala

```diff
@@ -116,7 +118,12 @@ object ABt {
           // Empty combiner += value
           createCombiner = (t: (Array[K], Array[Int], Matrix)) =>  {
             val (rowKeys, colKeys, block) = t
-            val comb = new SparseMatrix(prodNCol, block.nrow).t
```

Comment:
I see now that there is more to this than I'd originally thought.  We have to be careful in operations like `drmA %*% drmB` to have both sparse and dense blocks in the returned DRM.